docs/vi/index.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: defaut
+title: Học sâu
```

Comment:
Can you use all-caps?

Reply:
All-caps mean?

Reply:
ALL-CAPS is the all-caps version of "all-caps".
The website title is "DEEP LEARNING" not "Deep learning".

Reply:
Oh. I had changed this title.